common/presenters/move-to-card-component.js

```diff
@@ -9,6 +9,7 @@ function moveToCardComponent({
   showMeta = true,
   showTags = true,
   showStatus = true,
+  showIsALockout = false,
```

Comment:
I'm not sure we need this argument, since we set it to true if the move is a lockout anyway.

I think it's working differently to the other `show` arguments, where they decide whether or not to include this information in the component, whereas with `showIsALockout` if you set it to `false`, it'll show it regardless if the move is a lockout.

Reply:
Since we have no need to hide the lockout message, I think we shouldn't give people the option, at least for now.

Reply:
Ahh yes this makes sense, I forgot to update these tests to reflect the api change a little better